docs/managed-zone.md

```diff
@@ -83,5 +83,5 @@ At the moment the MGC is given credentials to connect to the DNS provider at sta
 There are plans to make this more customizable and dynamic in the future, [work tracked here](https://github.com/Kuadrant/multicluster-gateway-controller/issues/228).
 
 ## Spec of a ManagedZone
-The ManagedZone is a simple resource with an uncomplicated API, see a sample [here](../config/samples/kuadrant.io_v1alpha1_managedzone.yaml).
+The ManagedZone is a simple resource with an uncomplicated API, see a sample [here](https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/config/samples/kuadrant.io_v1alpha1_managedzone.yaml).
```

Comment:
@mikenairn yeah, other option is: we add the config path to the import paths for mkdocs, which will hover these resources up into the bundle. Download links will then download the file as is, from the branch you've selected from the import URL. That may be preferable as we seem to be heading towards versioning our docs in tandem with component releases.

Reply:
Right i see, we would need to update this i guess to include all relatively linked files here? https://github.com/Kuadrant/docs.kuadrant.io/blob/main/mkdocs.yml#L52

Bit annoying maybe, but probably better option long term. If we have the PR builds check with `mkdocs build -s` we should at least be able to catch and fix them as we update versions before we hit main of docs.

Reply:
Exactly yes

Thinking on it more, I think that might be our best route. The associated docs PR for this will build mkdocs with the `-s` switch (when these broken link PRs land)